strings/src/main/res/values/strings.xml

```diff
@@ -670,7 +670,7 @@
     <string name="save_legacy_settings_warning">Will be removed in a future version. Please use the QR code above.</string>
     <string name="reason_for_changes_description">You need to explain the reason for changes to this form</string>
 
-    <string name="scoped_storage_banner_text">You must migrate Collect forms to private storage to comply with Android requirements. In Collect v1.29, <b>the migration will be forced on install</b>.</string>
+    <string name="scoped_storage_banner_text">You must migrate Collect forms to private storage to comply with Android requirements. The migration should have happened automatically. If you still see this message it indicates that something went wrong. Please try to perform migration manually.</string>
```

Comment:
"The migration should have happened automatically. If you still see this message it indicates that something went wrong. Please try to perform migration manually." -> "An automatic migration attempt failed. Please migrate manually."

Reply:
Done. Do you also want to add something to the message that is displayed when migration is performed without errors? Now it's just `Storage migration completed!` but I remember that you were thinking about it, that it might be unclear for users that don't perform it manually.

Reply:
I think it's ok. Maybe a toast would be less disruptive since it's not really all that important for users to see. Let's get to beta and ask for feedback.